wrphttp/decoders_test.go

```diff
@@ -40,15 +40,22 @@ func testDecodeEntitySuccess(t *testing.T) {
 		defaultFormat wrp.Format
 		bodyFormat    wrp.Format
 		contentType   string
+		accept        string
+		name          string
 	}{
-		{wrp.Msgpack, wrp.Msgpack, ""},
-		{wrp.JSON, wrp.JSON, ""},
-		{wrp.Msgpack, wrp.JSON, wrp.JSON.ContentType()},
-		{wrp.JSON, wrp.Msgpack, wrp.Msgpack.ContentType()},
+		// Default test cases
+		{wrp.Msgpack, wrp.Msgpack, "", "", "default WRP headers"},
+		{wrp.JSON, wrp.JSON, "", "", "default JSON headers"},
+		// Accept header test cases
+		{wrp.JSON, wrp.JSON, "", wrp.JSON.ContentType(), "JSON Accept header"},
+		{wrp.JSON, wrp.JSON, "", wrp.Msgpack.ContentType(), "WRP Accept header"},
+		// Content-Type header test cases
+		{wrp.Msgpack, wrp.JSON, wrp.JSON.ContentType(), "", "JSON Content-Type header"},
+		{wrp.JSON, wrp.Msgpack, wrp.Msgpack.ContentType(), "", "WRP Content-Type header"},
 	}
 
-	for i, record := range testData {
-		t.Run(strconv.Itoa(i), func(t *testing.T) {
+	for _, record := range testData {
+		t.Run(fmt.Sprintf("test case: %v", record.name), func(t *testing.T) {
```

Comment:
```suggestion
		t.Run(record.name, func(t *testing.T) {
```
These are all test cases, so "test case" doesn't add any new information. 🙂